rifle.py

```diff
@@ -95,35 +94,10 @@ def update(self, events):
     """
 
     def draw(self, screen):
-        self.draw_background(screen)
-        self.draw_grid(screen)
         self.draw_entities(screen)
         self.draw_target_tags(screen)
         self.targeting_terminal_.draw_terminal(screen)
 
-    def draw_background(self, screen):
-        pygame.draw.rect(screen, BLACK, pygame.Rect((0, 0), screen.get_size()))
-
-    def draw_grid(self, screen):
```

Comment:
So I think you broke the main.py file (you must be running only the terminalfighter.py). The grid no longer shows up on the main menu and the bullet doesn't get erased when it travels through space

Reply:
fixed